modules/iam-role-for-service-accounts-eks/policies.tf

```diff
@@ -1446,3 +1446,72 @@ resource "aws_iam_role_policy_attachment" "vpc_cni" {
   role       = aws_iam_role.this[0].name
   policy_arn = aws_iam_policy.vpc_cni[0].arn
 }
+
+################################################################################
+# Amazon CloudWatch Observability Policy
+################################################################################
+
+data "aws_iam_policy_document" "amazon_cloudwatch_observability" {
+  count = var.create_role && var.attach_cloudwatch_observability_policy ? 1 : 0
+
+  statement {
+    sid       = "CollectEBSVolumeIDs"
+    actions   = ["ec2:DescribeVolumes"]
```

Comment:
since this is already covered in the `CloudWatchAgentServerPolicy` managed policy - we can actually just use the two managed policies as is without a custom policy